plugins/modules/nios_network.py

```diff
@@ -238,13 +238,15 @@ def check_ip_addr_type(obj_filter, ib_spec):
         check_ip = ip.split('/')
         del ib_spec['container']  # removing the container key from post arguments
         del ib_spec['options']  # removing option argument as for network container it's not supported
+        del ib_spec['template'] # removing template argument as it is not searchable argument
```

Comment:
Add a space before comment to fix sanity issue. There should be 2 spaces before a comment.
`del ib_spec['template']  # removing template argument as it is not searchable argument`